app/manage/authz.js

```diff
@@ -59,20 +59,19 @@ async function acceptToken (token, res, next) {
   if (result && result.active) {
     res.locals.user_id = result.sub
     return next()
-  }
-  else {
-    // Delete this accessToken ? 
+  } else {
+    // Delete this accessToken ?
     return res.status(401).send('Access denied, expired token')
   }
 }
 
 /**
- * Protected routes first check the session for the user,
- * Get the associated accessToken from the database, and check for 
+ * Routes with `authenticate` middleware first check the session for the user,
+ * Get the associated accessToken from the database, and check for
  * the accessToken validity with hydra. If there isn't a session,
  * it checks for an Authorization header with a valid access token
  */
-async function protected (req, res, next) {
+async function authenticate (req, res, next) {
```

Comment:
Apparently `protected` is a "protected" keyword so we're not supposed to use it.